src/query/service/tests/it/configs.rs

```diff
@@ -142,6 +142,9 @@ root = ""
 [catalog]
 meta_store_address = "127.0.0.1:9083"
 protocol = "binary"
+
+[iam]
```

Comment:
`iam` in config seems confusing.

What happened if users want to deploy databend on azure?

Reply:
you are right, although it is in fact a general abbr. https://en.wikipedia.org/wiki/Identity_management

Do you have any suggestions?

- users: it may include other things.
- security: should include TLS configs too

Reply:
> How about `idm`? (Although it may confuse with `Internet Download Manager`)
> 
> `identities` also LGTM.

Ok, I prefer `identities`